### PR TITLE
refactor(website-example): performance, image and seo improvements

### PR DESCRIPTION
**Pull request by Itrulia**
A few bug fixes too

### CODE REVIEW

File: apps/website-example/next.config.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 //@ts-check
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 const {composePlugins, withNx} = require('@nx/next')
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
```
Comment (Itrulia) [inline apps/website-example/next.config.js:4]: Allows us to see the bundle analyzer for prod builds

File: apps/website-example/public/robots.txt
Comment (Itrulia) [inline apps/website-example/public/robots.txt:1]: for SEO

File: libs/author/website/src/lib/author-list/author-list-item.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ export function AuthorListItem({
   } = useWebsiteBuilder()
 
   return (
-    <Link css={wrapperStyles(theme)} className={className} href={slug}>
```
Comment (Itrulia) [inline libs/author/website/src/lib/author-list/author-list-item.tsx:40]: Bug fix as it was not pointing to the correct url

File: libs/block-content/website/src/lib/facebook/facebook-video-block.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import styled from '@emotion/styled'
 import {Block, FacebookVideoBlock as FacebookVideoBlockType} from '@wepublish/website/api'
 import {BuilderFacebookVideoBlockProps} from '@wepublish/website/builder'
-import ReactPlayer from 'react-player'
```
Comment (Itrulia) [inline libs/block-content/website/src/lib/facebook/facebook-video-block.tsx:4]: smaller bundle

File: libs/feed/website/src/lib/feed-generator.ts [hunk 4]
```diff
@@ -48,14 +48,15 @@ export const generateFeed =
       } as Item
     })
 
+    const Feed = (await import('feed')).Feed
```
Comment (Itrulia) [inline libs/feed/website/src/lib/feed-generator.ts:51]: The feed library is quite big and somehow webpack isn't able to properly treshake it out on the client. This fixes it.

File: libs/image/website/src/lib/image.tsx [hunk 5]
```diff
@@ -24,10 +24,11 @@ export const imageToSquareImageItems = (image: FullImageFragment): ImageItem[] =
   {url: image.squareSmallURL, size: 200}
 ]
 
-export const ImageWrapper = styled('img')`
+export const ImageWrapper = styled('img')<{aspectRatio: number}>`
   max-width: 100%;
   width: 100%;
   height: auto;
+  aspect-ratio: auto ${({aspectRatio}) => aspectRatio};
```
Comment (Itrulia) [inline libs/image/website/src/lib/image.tsx:31]: Fixes image going haywire when the image can't load. As aspect-ratio by the user agent is only set when the image is loaded.

File: libs/richtext/src/lib/to-html.tsx [hunk 6]
```diff
@@ -1,11 +1,12 @@
 import {Node} from 'slate'
-import {slateToDomConfig, slateToHtml} from 'slate-serializers'
 import {BlockFormat} from './block-format'
 import {InlineFormat} from './inline-format'
 import {TextFormat} from './text-format'
 
-export const toHtml = (nodes: Node[]) =>
-  slateToHtml(nodes, {
+export const toHtml = async (nodes: Node[]) => {
+  const {slateToHtml, slateToDomConfig} = await import('slate-serializers')
```
Comment (Itrulia) [inline libs/richtext/src/lib/to-html.tsx:7]: slate-serializers is quite a big package, no need to load it on every page even when this function isn't called.

File: libs/image/website/src/lib/image.tsx (hunk 5 above)
Comment (antkiewiczk) [reply libs/image/website/src/lib/image.tsx:31]: shouldn't there be a default?

File: libs/block-content/website/src/lib/facebook/facebook-video-block.tsx (hunk 3 above)
Comment (antkiewiczk) [reply libs/block-content/website/src/lib/facebook/facebook-video-block.tsx:4]: 👍

File: libs/website/builder/src/lib/website-builder.context.tsx [hunk 7]
```diff
@@ -173,7 +173,7 @@ export type WebsiteBuilderProps = {
   }
 
   date: {
-    format: (date: Date) => string
+    format: (date: Date, includeTime?: boolean) => string
```
Comment (antkiewiczk) [inline libs/website/builder/src/lib/website-builder.context.tsx:176]: 👍

**Review by antkiewiczk — APPROVED** [review]

File: libs/image/website/src/lib/image.tsx (hunk 5 above)
Comment (Itrulia) [reply libs/image/website/src/lib/image.tsx:31]: This is the default

**Pull request merged**